src/Thrift.Net.Compiler/Program.cs

```diff
@@ -58,50 +58,62 @@ public class Program
         /// </returns>
         public static int Main(FileInfo input, DirectoryInfo outputDirectory, IConsole console = null)
         {
-            if (!input.Exists)
+            try
             {
-                console.Error.Write($"The specified input file '{input.Name}' could not be found.{Environment.NewLine}");
-                return (int)ExitCode.InputFileNotFound;
-            }
+                if (!input.Exists)
```

Comment:
Maybe worth extracting this code out to a method, just to keep the `Main()` method fairly simple?